nix/configuration.nix

```diff
@@ -76,6 +76,11 @@ in
       default = true;
     };
 
+    concrexit.rejectUnknownHost = mkOption {
+      type = types.bool;
+      default = cfg.ssl;
```

Comment:
If this is the default, the option should be named `allowUnknownHost` or the logic should be reversed. Just run through it and you'll see what I mean

Reply:
Also I recommend to add a description to this option